api/valor_api/backend/core/llm_instructions_analysis.py

```diff
@@ -0,0 +1,573 @@
+def _generate_claims_instruction(text: str) -> str:
+    """
+    Instruction template was adapted from DeepEval's codebase https://github.com/confident-ai/deepeval/blob/main/deepeval/metrics/faithfulness/template.py.
```

Comment:
in these docstrings, can you start by describing what this function does / why it's needed?